Testscripts/Linux/utils.sh

```diff
@@ -3509,7 +3509,7 @@ function check_package ()
 				;;
 
 			ubuntu|debian)
-				apt-cache policy "$package_name" | grep Candidate
+				apt-cache policy "$package_name" | grep "Candidate" | grep -v "none"
```

Comment:
Any explain?

Reply:
> Any explain?
In EOAN, below package not existed, but use previous code, it returns 0, add one more strict, won't affect previous functionality
EOAN
```
apt-cache policy "btrfs-tools" | grep "Candidate"
  Candidate: (none)
echo $?
0
apt-cache policy "btrfs-tools" | grep "Candidate" | grep -v "none"
echo $?
1
```
BIONIC
```
apt-cache policy "GIT" | grep "Candidate" | grep -v "none"
echo $?
1
apt-cache policy "git" | grep "Candidate" | grep -v "none"
  Candidate: 1:2.17.1-1ubuntu0.4
echo $?
0
```